test/src/utils/test-utils.js

```diff
@@ -11,8 +11,10 @@ function getConfigStructure() {
       adBreaks: []
     },
     sources: {},
+    text: {
+      enableCEA708Captions: true
+    },
     playback: {
-      enableCEA708Captions: true,
```

Comment:
Maybe leave this test to check backward compatibility and add a comment to mention that?

Reply:
Oh, but its one config for all tests... Nevermind